packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -203,46 +181,8 @@ export class OdspDocumentService implements IDocumentService {
      * @returns returns the document delta storage service for sharepoint driver.
      */
     public async connectToDeltaStorage(): Promise<IDocumentDeltaStorageService> {
-        const snapshotOps = this.storageManager?.ops ?? [];
-        const service = new OdspDeltaStorageService(
-            this.odspResolvedUrl.endpoints.deltaStorageUrl,
-            this.getStorageToken,
-            this.epochTracker,
-            this.mc.logger,
-        );
-
-        // batch size, please see issue #5211 for data around batch sizing
-        const batchSize = this.hostPolicy.opsBatchSize ?? 5000;
-        const concurrency = this.hostPolicy.concurrentOpsBatches ?? 1;
-        return new OdspDeltaStorageWithCache(
-            snapshotOps,
-            this.mc.logger,
-            batchSize,
-            concurrency,
-            async (from, to, telemetryProps, fetchReason) => service.get(from, to, telemetryProps, fetchReason),
-            async (from, to) => {
-                const res = await this.opsCache?.get(from, to);
-                return res as ISequencedDocumentMessage[] ?? [];
-            },
-            (from, to) => {
-                if (this.currentConnection !== undefined && !this.currentConnection.disposed) {
-                    this.currentConnection.requestOps(from, to);
-                }
-            },
-            (ops: ISequencedDocumentMessage[]) => this.opsReceived(ops),
-        );
-    }
-
-    /** Annotate the given error indicating which connection step failed */
-    private annotateConnectionError(
-        error: any,
-        failedConnectionStep: string,
-        separateTokenRequest: boolean,
-    ): IFluidErrorBase {
-        return normalizeError(error, { props: {
-            failedConnectionStep,
-            separateTokenRequest,
-        } });
+        const service = await this.getDelayLoadedDocumentService();
```

Comment:
I think this will run on boot, at least for Loop, as leveraging IContainerLoadMode.opsBeforeReturn === "cached" will result in request to cached ops.
I've opened https://dev.azure.com/fluidframework/internal/_queries/edit/2567/?triage=true to attempt to make default path == recommended (and used by Bohemia) path, but at the moment we should assume in our bundle size tests that's the mode used.